Formula/youtubedr.rb

```diff
@@ -0,0 +1,34 @@
+class Youtubedr < Formula
+  desc "Download Youtube Video in Golang"
+  homepage "https://github.com/kkdai/youtube"
+  url "https://github.com/kkdai/youtube/archive/v2.2.1.tar.gz"
+  sha256 "e0e298aa4d589813b635bc09e9b50b220f248312ec5893a381bacf70a624a9e1"
+  license "MIT"
+
+  bottle :unneeded
```

Comment:
It does need a bottle

Reply:
sure, I will figure it out how to add a bottle

Reply:
You don't need to do anything, just don't specify that it doesn't need a bottle.

https://docs.brew.sh/Bottles#creation
> Bottles are created using the Brew Test Bot, usually when people submit pull requests to Homebrew. The bottle do block is updated by maintainers when they merge a pull request.